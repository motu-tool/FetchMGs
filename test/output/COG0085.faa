version https://git-lfs.github.com/spec/v1
oid sha256:9d7bf27d3ba4f7e1160928040f4adc9b3d53268b28c900661ff5caade0c50c63
size 10653
