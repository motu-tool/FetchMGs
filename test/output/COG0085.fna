version https://git-lfs.github.com/spec/v1
oid sha256:5d2441713e100e237a8d4fcb7dba3ca0da6b44475a8a15fe7c94f634c98e8822
size 29900
