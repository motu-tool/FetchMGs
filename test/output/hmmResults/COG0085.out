# hmmsearch :: search profile(s) against a sequence database
# HMMER 3.3.2 (Nov 2020); http://hmmer.org/
# Copyright (C) 2020 Howard Hughes Medical Institute.
# Freely distributed under the BSD open source license.
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# query HMM file:                  /nfs/home/fieldc/.local/lib/python3.7/site-packages/fetchmgs/data/COG0085.hmm
# target sequence database:        ../fetchMGs/example_datasets/example_data_genomes.faa
# output directed to file:         output/hmmResults/COG0085.out
# per-dom hits tabular output:     output/hmmResults/COG0085.dom
# show alignments in output:       no
# max ASCII text line length:      unlimited
# sequence reporting threshold:    score >= 1020
# number of worker threads:        1
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Query:       COG0085  [M=2571]
Scores for complete sequences (score includes all domains):
   --- full sequence ---   --- best 1 domain ---    -#dom-
    E-value  score  bias    E-value  score  bias    exp  N  Sequence           Description
    ------- ------ -----    ------- ------ -----   ---- --  --------           -----------
          0 2210.2  10.6          0 2210.0  10.6    1.0  1  1037409.BJ6T_43770  protein_id="BAL09646.1" gene="rpoB" product="DNA-directed RNA polymerase beta chain" GI="354956967" project_id="67463"
          0 2136.9  10.7          0 2136.7  10.7    1.0  1  1089456.NCGM2_5515  protein_id="BAK92336.1" gene="rpoB" product="DNA-directed RNA polymerase subunit beta" GI="348036976" note="similar to GI:116052305; similar to PA14_08760 of P. aeruginosa UCBPP-PA14" project_id="73453"
          0 2010.3   0.9          0 2010.0   0.9    1.0  1  1148.SYNGTS_0791    protein_id="BAK49539.1" gene="rpoB" product="DNA-directed RNA polymerase subunit beta" GI="339273052" note="ORF_ID:sll1787" project_id="67081"
          0 1982.0  21.8          0 1981.9  21.8    1.0  1  944546.ABED_1705    protein_id="BAK71422.1" product="DNA-directed RNA polymerase beta subunit" GI="345469971" project_id="61719"
          0 1978.0  21.6          0 1977.9  21.6    1.0  1  944547.ABLL_2469    protein_id="BAK74344.1" product="DNA-directed RNA polymerase beta subunit" GI="345472894" project_id="61721"
          0 1561.9   0.4          0 1561.6   0.4    1.0  1  944547.ABLL_2468    protein_id="BAK74343.1" product="DNA-directed RNA polymerase beta' subunit" GI="345472893" project_id="61721"
          0 1554.3   0.1          0 1554.0   0.1    1.0  1  944546.ABED_1704    protein_id="BAK71421.1" product="DNA-directed RNA polymerase beta' subunit" GI="345469970" project_id="61719"


Domain annotation for each sequence:
>> 1037409.BJ6T_43770  protein_id="BAL09646.1" gene="rpoB" product="DNA-directed RNA polymerase beta chain" GI="354956967" project_id="67463"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ! 2210.0  10.6         0         0      60    1121 ..       4    1363 ..       1    1372 [] 0.96

>> 1089456.NCGM2_5515  protein_id="BAK92336.1" gene="rpoB" product="DNA-directed RNA polymerase subunit beta" GI="348036976" note="similar to GI:116052305; similar to PA14_08760 of P. aeruginosa UCBPP-PA14" project_id="73453"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ! 2136.7  10.7         0         0      63    1117 ..       6    1356 ..       1    1357 [] 0.97

>> 1148.SYNGTS_0791  protein_id="BAK49539.1" gene="rpoB" product="DNA-directed RNA polymerase subunit beta" GI="339273052" note="ORF_ID:sll1787" project_id="67081"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ! 2010.0   0.9         0         0      80    1135 ..       8    1080 ..       3    1102 .] 0.95

>> 944546.ABED_1705  protein_id="BAK71422.1" product="DNA-directed RNA polymerase beta subunit" GI="345469971" project_id="61719"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ! 1981.9  21.8         0         0      62    1122 ..       5    1380 ..       1    1382 [] 0.94

>> 944547.ABLL_2469  protein_id="BAK74344.1" product="DNA-directed RNA polymerase beta subunit" GI="345472894" project_id="61721"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ! 1977.9  21.6         0         0      62    1120 ..       5    1378 ..       1    1381 [] 0.94

>> 944547.ABLL_2468  protein_id="BAK74343.1" product="DNA-directed RNA polymerase beta' subunit" GI="345472893" project_id="61721"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ! 1561.6   0.4         0         0    1141    2557 ..      91    1504 ..      11    1508 .] 0.98

>> 944546.ABED_1704  protein_id="BAK71421.1" product="DNA-directed RNA polymerase beta' subunit" GI="345469970" project_id="61719"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ! 1554.0   0.1         0         0    1141    2561 ..      91    1508 ..      10    1511 .] 0.98



Internal pipeline statistics summary:
-------------------------------------
Query model(s):                            1  (2571 nodes)
Target sequences:                      23272  (7230323 residues searched)
Passed MSV filter:                      1297  (0.0557322); expected 465.4 (0.02)
Passed bias filter:                      429  (0.0184342); expected 465.4 (0.02)
Passed Vit filter:                        56  (0.00240633); expected 23.3 (0.001)
Passed Fwd filter:                        13  (0.000558611); expected 0.2 (1e-05)
Initial search space (Z):              23272  [actual number of targets]
Domain search space  (domZ):               7  [number of targets reported over threshold]
# CPU time: 1.79u 0.09s 00:00:01.88 Elapsed: 00:00:01.76
# Mc/sec: 10531.01
//
[ok]
